Source/WebKit/NetworkProcess/webtransport/NetworkTransportReceiveStream.h

```diff
@@ -49,6 +49,8 @@ class NetworkTransportReceiveStream : public RefCounted<NetworkTransportReceiveS
 public:
     template<typename... Args> static Ref<NetworkTransportReceiveStream> create(Args&&... args) { return adoptRef(*new NetworkTransportReceiveStream(std::forward<Args>(args)...)); }
 
+    virtual ~NetworkTransportReceiveStream() { }
```

Comment:
`= default`?

Reply:
Yes, we should use `default`, but also this change cause a build failure with upstream clang because `m_identifier` and `m_session` are not fully defined for the destructor.

Please move the destructor to the `NetworkTransportReceiveStream.cpp` source file.

See <https://commits.webkit.org/282753@main> for details.